### PR TITLE
[USH-1679] Rename the heading of the new post screen to 'Add New Post'

### PR DESCRIPTION
**Pull request by MMwandigha**
Renaming the heading of the new post breadcrumb to 'Add New Post'

**Testing**
- [ ] Open https://mzima-dev.staging.ush.zone/map
- [ ] Click on 'Add New Post'
- [ ] Observe the heading/breadcrumb text at the top

Previously it was 'Create Post' but now it's 'Add New Post'

### CODE REVIEW

**Review by Ifycode — APPROVED** [review]
🎉

File: apps/web-mzima-client/src/app/post/post-routing.module.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const routes: Routes = [
   {
     path: 'create/:type',
     component: PostEditComponent,
-    data: { breadcrumb: 'Create Post' },
+    data: { breadcrumb: 'Add New Post' },
```
Comment (Ifycode) [inline apps/web-mzima-client/src/app/post/post-routing.module.ts:15]: We can just have another ticket to add translation for the breadcrumb text in this file, similar to how it's done in app routing module file @MMwandigha

Comment (MMwandigha) [reply apps/web-mzima-client/src/app/post/post-routing.module.ts:15]: @Ifycode Oh yeah its true... I also thought of the same. I noticed that all the breadcrumb texts in the post-routing module haven't been translated yet. I shall create one

Comment (Ifycode) [reply apps/web-mzima-client/src/app/post/post-routing.module.ts:15]: 👍🏼

Comment (MMwandigha) [reply apps/web-mzima-client/src/app/post/post-routing.module.ts:15]: Here's the associated PR for this:
https://github.com/ushahidi/platform-client-mzima/pull/1483

**Pull request merged**